index.html

```diff
@@ -200,6 +200,48 @@ <h2>About Myself</h2>
           />
         </ul>
       </section>
+      <section class="contactme">
+        <h1 class="contacttxt font">Contact me</h1>
+        <p class="paragraph font">
```

Comment:
- [ ] Kindly change this to an `h2-6` instead of `h1`. As a best practice, you'll have to make sure to use only one `h1` for the page which is the main title, and then each title after that should follow the title hierarchy.

Reply:
Done